README.md

```diff
@@ -27,6 +27,11 @@ If you're interested developing this application, follow these steps to get runn
 
 You will now be able to develop the frontend React application. Any changes made to the code will hot-reload upon save.
 
+Failing to compile on a windows environment?
+Run the following commands and then retry step 4:
+`git fetch --all`
+`git reset --hard`
+
```

Comment:
Note: If the above is failing to compile on a Windows machine, you will first need to run `git fetch --all` and then `git reset --hard`. After successfully running these two commands, you can then retry step 4 (`npm start`).